services/web/server/src/simcore_service_webserver/catalog.py

```diff
@@ -80,7 +80,7 @@ def setup_catalog(app: web.Application, *, disable_auth=False):
     # ----------------------------------------------
     # TODO: temporary, just to check compatibility between
     # trafaret and pydantic schemas
-    cfg = assert_valid_config(app).copy()
+    cfg, _ = assert_valid_config(app)
```

Comment:
underscore is convention for an unimportant return value which will not be used but discarded, correct?

Reply:
yes. 

in any case, just to clarify" all the ``assert_valid_config`` is a tmp check taht will be removed as soon as the old config version is dropped